src/galileo/resources/api/datasets/bulk_delete_datasets_datasets_bulk_delete_delete.py

```diff
@@ -26,6 +27,8 @@ def _get_kwargs(*, body: BulkDeleteDatasetsRequest) -> dict[str, Any]:
 
     headers["Content-Type"] = "application/json"
 
+    headers["X-Galileo-SDK"] = f"galileo-python/{get_package_version()}"
```

Comment:
Did you add this manually?

Reply:
Ok I see it's in the jinja template.. interesting that it was missing from this file and not the rest, I guess doesn't matter 🤷🏾‍♂️

Reply:
But actually yeah I haven't checked every file.. github is forcing me to look at each file one at a time 😂